website/docs/r/msk_cluster.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: "aws"
+page_title: "AWS: aws_msk_cluster"
+sidebar_current: "docs-aws-resource-msk-cluster"
```

Comment:
This documentation page is missing a sidebar link in a new `MSK Resources` section of `website/aws.erb`